.github/workflows/at_server.yaml

```diff
@@ -252,7 +261,7 @@ jobs:
 
       - name: Run docker container
         # -d: run container in detached mode. --rm: remove container on stop -p: bind ports to host
-        run: docker run -d --rm --name at_virtual_env_cont -e testingMode="true" -p 6379:6379 -p 25000-25019:25000-25019 -p 64:64 at_virtual_env:trunk
+        run: docker run -d --rm --name at_virtual_env_cont -e testingMode="true" -e httpsEnabled="true" -p 6379:6379 -p 25000-25099:25000-25099 -p 64:64 -p 443:443 at_virtual_env:trunk
 
       - name: Check docker readiness to load PKAM keys
```

Comment:
Changed only the at_functional_tests job so it builds and copies the `root` binary into the newly-built ve image ... this stuff does change very infrequently so on the occasions we do need to get a new `root` into ee and ve, we can rebuild ve_base, ee and ve